script_01.py

```diff
@@ -79,12 +79,89 @@ def task6(a=[1,2,12,4], b=[2,4,2,9]):
     else:
         print('Multipication impossible')
 
-# def task7()
+def task7(n=8):
+    """task7 generates a dictionary of n pairs which are (i, i*i)
+    and prints it"""
+    dictionary=dict()
+    for i in range (1, n+1):
+        dictionary[i] = i*i
+    print(dictionary)
+
+def task8(to_replace='two 755 ff'):
+    """Function task8 converts numbers in variable to their text
+    equivalents. If there is already such text equivalent, Function
+    leaves it there."""
+    numbers = {
+    '0' : "zero",
+    '1' : "one",
+    '2' : "two",
+    '3' : "three",
+    '4' : "four",
+    '5' : "five",
+    '6' : "six",
+    '7' : "seven",
+    '8' : "eight",
+    '9' : "nine"
+    }
+
+    a_length = str(to_replace)
+    text = ''
+    list_elements = a_length.split()
+
+    for n in range (0, len(list_elements)):
+        if list_elements[n] in numbers.values():
+            text = text + ' ' + str(list_elements[n])
+
+        else:
+            for i in range (0, len(list_elements[n])):
+                if list_elements[n][i] in numbers:
+                    text = text + ' ' + str(numbers[list_elements[n][i]])
+
+    text = text.strip()
```

Comment:
why do you use `strip()`?

Reply:
It removes the space in the begging of the text

Reply:
+1